DashKit/DashPumpManagerError.swift

```diff
@@ -32,6 +33,17 @@ public enum DashPumpManagerError: Error, LocalizedError {
             return error.errorDescription
         case .busy:
             return LocalizedString("Pod Busy.", comment: "Description of DashPumpManagerError error when pump manager is busy.")
+        case .acknowledgingAlertFailed:
+            return LocalizedString("Tidepool Loop was unable to clear the alert on your Pod, therefore you may continue to hear an audible beep.", comment: "Description of DashPumpManagerError error when acknowledging alert failed.")
+        }
+    }
+    
+    public var recoverySuggestion: String? {
```

Comment:
Where is this referenced?  Is it in a later PR?

Reply:
Oops, yeah, here's the Loop PR: https://github.com/tidepool-org/Loop/pull/435